### DISCUSSION TITLE
Pixelorama preventing mac to sleep

### DISCUSSION THREAD

**Discussion by msemelman** [Orama-Interactive/Pixelorama#1124] [Q&A] [answered]
I'm using pixelorama 1.0.3-stable and it prevents mac from sleeping.
<img width="926" alt="image" src="https://github.com/user-attachments/assets/14ee9be6-0bf7-4941-9c51-ec35bc06c66f">
Any idea why?

**Comment by msemelman** [comment]
According to `pmset -g assertions` command:
```Listed by owning process:
   pid 10707(Pixelorama): [0x000160ed00058f64] 00:19:50 PreventUserIdleDisplaySleep named: "Pixelorama"
	Details: Godot Engine running with display/window/energy_saving/keep_screen_on = true```

**Comment by Variable-ind** [comment]
This may be made as a changeable option in the preference dialog

**Comment by msemelman** [comment]
I agree that being a setting in preferences can be useful. 
I would encourage that such setting is false by default as normally other editors don't keep screen on 
Users normally use a system wide setting or apps to achieve such behavior.

Please check my proposal, it may serve as reference or initial solution.
https://github.com/Orama-Interactive/Pixelorama/pull/1125

**Comment by msemelman** [comment, marked as answer]
Thank you, version 1.0.4 fixes it!
![image](https://github.com/user-attachments/assets/5ab524a0-6ba4-46e4-967c-d728d6012873)
